src/current/cockroachcloud/ccloud-get-started.md

```diff
@@ -118,13 +119,13 @@ Use the `ccloud cluster create` command to create a new CockroachDB {{ site.data
 ccloud cluster create dedicated
 ~~~
 
-This command creates a 1 node CockroachDB {{ site.data.products.dedicated }} cluster with 2 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
+This command creates a 3-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```

Comment:
```suggestion
This command creates a 1-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```

The `ccloud` commands have not yet been updated. I'm not sure if the tier names are being updated or we are going with small-s serverless and small-d dedicated.